docs/reference/remap/assert.cue

```diff
@@ -0,0 +1,38 @@
+package metadata
+
+remap: functions: assert: {
+	arguments: [
+		{
+			name:        "condition"
+			description: "The condition to check."
+			required:    true
+			type: ["boolean"]
+		},
+		{
+			name:        "message"
+			description: "Should condition be false, message will be reported as the failure message."
+			required:    true
```

Comment:
It would be nice if we could make this optional, and have a fairly useful auto-generated error message.

Something like:

```rust
assert(true == false)
// assertion failed: true == false
```